hbase-backup/src/main/java/org/apache/hadoop/hbase/backup/master/LogRollMasterProcedureManager.java

```diff
@@ -171,7 +199,20 @@ private boolean isBackupEnabled() {
   }
 
   @Override
-  public boolean isProcedureDone(ProcedureDescription desc) {
-    return done;
+  public boolean isProcedureDone(ProcedureDescription desc) throws IOException {
```

Comment:
Mind explaining a bit here?

So at upper layer, we do not know whether you are using the new procedure or the old 'procedure', and based on different implementation, we have different ways to determine whether it is finished.

Where do we put the ROLLLOG_PROCEDURE_ID into the ProcedureDescription? I haven't found the code.

Reply:
Ok. I'll add some comments here. Thanks Duo.